libs/community/langchain_community/llms/huggingface_hub.py

```diff
@@ -127,7 +127,7 @@ def _call(
                 response = hf("Tell me a joke.")
         """
         _model_kwargs = self.model_kwargs or {}
-        parameters = {**_model_kwargs, **kwargs}
+        parameters = {**_model_kwargs, **kwargs, "return_full_text": False}
```

Comment:
Isn't the current version more versatile? The user can pass `return_full_text` as a kwarg or as a model_kwarg to force this behavior if they need it.

Reply:
return_full_text should be the default to let it behave like other **langchain** language models